src/command_modules/azure-cli-container/azure/cli/command_modules/container/tests/test_container_commands.py

```diff
@@ -3,15 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+# pylint: disable=line-too-long
+
 from azure.cli.testsdk import (ScenarioTest, JMESPathCheck, JMESPathCheckExists, ResourceGroupPreparer)
 
 
 class AzureContainerInstanceScenarioTest(ScenarioTest):
+    # Test create container with image, os type, ip address type, port, cpu, memory, command line and evnironment variable specified.
```

Comment:
I would agree that this comment is long.
Also, there's a typo with 'evnironment'.